lib/workers/repository/update/branch/schedule.ts

```diff
@@ -112,8 +112,11 @@ function cronMatches(cron: string, now: DateTime): boolean {
   return true;
 }
 
-export function isScheduledNow(config: RenovateConfig): boolean {
-  let configSchedule = config.schedule;
+export function isScheduledNow(
+  config: RenovateConfig,
+  schedule: string[]
```

Comment:
pass the key instead, as we already have the config object
```suggestion
  scheduleKey: 'schedule' | ' automergeSchedule' = 'schedule'
```

with fallback to have less pr changes